Js/index.js

```diff
@@ -4,24 +4,175 @@ let btnCancelMobileMenu = document.querySelector("#btn-close-nav");
 const PortfolioClicked = document.querySelector(".link-portfolio");
 const AboutClicked = document.querySelector(".link-about");
 const ContactClicked = document.querySelector(".link-contact");
+const allBtnPopups = document.querySelectorAll(".btn-pop");
```

Comment:
Hi @rebel216 
Your app works very well, just one important issue you need to fix as requested by the previous reviewer.
Please add a cross(X) button to close the hamburger menu.
Please kindly fix this issue if not the user will find it difficult to return to the home page after clicking on the `See Project` button 
![closs](https://user-images.githubusercontent.com/55596970/167953673-3418e882-0c0f-4fdc-86e2-2df7d4c3fd0f.png)

Reply:
Done 👍